openshift_tools/cloud/aws/ebs_snapshotter.py

```diff
@@ -272,6 +276,7 @@ def get_expired_snapshots(snapshots, hourly_backups, daily_backups, weekly_backu
         temp = []
 
         for t in target_backup_times:
+            # pylint: disable=singleton-comparison
```

Comment:
Please add a reason block here per our best practices:
https://github.com/openshift/openshift-tools/blob/prod/docs/best_practices_guide.adoc#All-PyLint-rule-disables-MUST-be-documented-in-the-code